site/ru/blog/new-in-devtools-94/index.md

```diff
@@ -0,0 +1,181 @@
+
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/XVUY8XxbGZW74kPsGkOZ.png", alt="Правильная трассировка стека ошибок для встроенных скриптов с #sourceURL", width="800", height="425" %}
+
+{# https://chromium.googlesource.com/v8/v8/+/c2f30c2b3f637c2339e8b9672c5c59a21b7d1095 #}
+
+Задача в трекере Chromiums: [1183990](https://crbug.com/1183990), ​​[578269](https://crbug.com/578269)
+
+## Изменение формата цвета во вкладке Computed {: #color-unit }
+
+Теперь вы можете изменить формат цвета у любого элемента прямо из вкладки Computed, зажав <kbd>Shift</kbd> и кликнув на превью цвета.
+
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/IhOkF5do9P8Ovlr7YsdX.png", alt="Shift+клик на превью цвета для изменения формата", width="800", height="474" %}
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/97143f7586d540e53a2e40ced7f106181e5c9ce3 #}
+
+Задача в трекере Chromium: [1226371](https://crbug.com/1226371)
+
+## Кастомные всплывающие подсказки заменены на нативные HTML-подсказки {: #tooltip }
+
+DevTools теперь поддерживает нативные HTML-подсказки во всех компонентах. В DevTools долгое время использовалась кастомная реализация всплывающих подсказок из-за невозможности стилизовать стандартные HTML-подсказки.
+
+К сожалению, поддержка кастомных всплывающих подсказок очень сложно и мы регулярно сталкивались со разными багами.
+
+Взвесив все преимущества кастомной реализации мы пришли к выводу, что для DevTools достаточно нативных HTML-подсказок. Их использование позволяет избежать множества проблем для наших пользователей.
+
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/bOFfHPAwX3qiVcgANPmh.png", alt="всплывающая подсказка в DevTools", width="800", height="452" %}
+
+{# https://chromium-review.googlesource.com/c/devtools/devtools-frontend/+/3008794 #}
+
+Задача в трекере Chromium: [1223391](https://crbug.com/1223391)
+
+
+## [Эксперимент] Скрытие задач на вкладке Issues {: #hide-issues }
+
+{% Aside %}
+Чтобы включить эту экспериментальную возможность, перейдите в **Settings** > **Experiments** и включите чекбокс **Enable hide issues menu**.
+{% endAside %}
+
+Включите эксперимент **hide issues menu** чтобы скрывать задачи во вкладке **Issues**. Это способ сосредоточиться только на важных для вас задачах.
+
+На вкладке **Issue** наведите курсор на задачу, кликните на иконку меню  &nbsp; {% Img src="image/admin/4sdCQbpBaG4MpoHB1J08.png", alt="More", width="4", height="20" %} &nbsp; справа и выберите **Hide issues like this** чтобы скрыть её.
+
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/GGJzvwvMYSrkirU44STQ.png", alt="Экспериментальное меню для скрытия задач", width="800", height="494" %}
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/0200fc96fecec0e209e84c21359ab53393860978 #}
+
+Задача в трекере Chromium: [1175722](https://crbug.com/1175722)
+
+{% include 'partials/devtools/ru/reach-out.md' %}
+{% include 'partials/devtools/ru/whats-new.md' %}
```

Comment:
please help to translate the content in these two files too!

Reply:
Done! 🙌